recipes/ffmpeg/all/conanfile.py

```diff
@@ -642,11 +642,12 @@ def opt_append_disable_if_set(args, what, v):
         ranlib = buildenv_vars.get("RANLIB")
         if ranlib:
             args.append(f"--ranlib={unix_path(self, ranlib)}")
-        # for some reason pkgconf from conan can't find .pc files on Linux in the context of ffmpeg configure...
-        if self._settings_build.os != "Linux":
-            pkg_config = self.conf.get("tools.gnu:pkg_config", default=buildenv_vars.get("PKG_CONFIG"), check_type=str)
-            if pkg_config:
-                args.append(f"--pkg-config={unix_path(self, pkg_config)}")
+        pkg_config = self.conf.get("tools.gnu:pkg_config", default=buildenv_vars.get("PKG_CONFIG"), check_type=str)
```

Comment:
@memsharded  - I'm not entirely sure about  buildenv_vars.get("PKG_CONFIG") - I suppose that covers when a user has the variable defined in the `[buildenv]` section, but not if a user has it in the actual environment (without having it in the profile)